va_explorer/users/views.py

```diff
@@ -56,6 +57,12 @@ class UserDetailView(CustomAuthMixin, UserDetailViewMixin, DetailView):
     login_url = reverse_lazy("account_login")
     model = User
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['va_usernames'] = VaUsername.objects.filter(user_id=self.object.id)
```

Comment:
Since it's a relation to user, you could do `user.vausername_set.all` in the template without needing to set this context variable.